fabtests/man/man7/fabtests.7

```diff
@@ -1,6 +1,6 @@
 .\" Automatically generated by Pandoc 1.19.2.4
 .\"
-.TH "fabtests" "7" "2020\-12\-01" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
+.TH "fabtests" "7" "2023\-07\-24" "Libfabric Programmer\[aq]s Manual" "#VERSION#"
```

Comment:
There's a missing github action somewhere that should be updating this file automatically, similar to the libfabric man pages.